setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
It looks like there is also a checkbox to "Install Project" in the admin tab on ReadTheDocs.  This appears to be selected for PyWavelets:

<img width="840" alt="screen shot 2017-04-03 at 6 29 16 pm" src="https://cloud.githubusercontent.com/assets/6528957/24634366/953d2354-189b-11e7-8561-e7b33f1d4404.png">